core/templates/dev/head/pages/exploration_editor/exploration_editor.html

```diff
@@ -332,6 +333,9 @@
   <script src="/templates/dev/head/pages/state_editor/StateEditorService.js"></script>
   <script src="/templates/dev/head/pages/state_editor/StatePropertyServices.js"></script>
 
+  <script src="/templates/dev/head/components/CkEditorRteDirective.js"></script>
+  <script src="/templates/dev/head/components/CkEditorWidgetsInitializer.js.js"></script>
```

Comment:
Please make sure to proofread your PRs (do a self-review of all files in the Files Changed tab immediately after you push).

Reply:
sorry about that.

Reply:
done.
PTAL